src/NUnitFramework/framework/Internal/PlatformHelper.cs

```diff
@@ -319,6 +322,18 @@ private bool IsRuntimeSupported(RuntimeType runtime, string versionSpecification
 
             return _rt.Supports(target);
         }
+
+        private bool IsNetCoreRuntimeSupported(RuntimeType runtime, string versionSpecification)
+        {
+            if (versionSpecification != null)
+            {
+                throw new PlatformNotSupportedException("Detection versions of .NET Core is not supported - " + runtime.ToString() + "-" + versionSpecification);
```

Comment:
`Detection` → `Detecting`

(If you like, you can use a C# 6 interpolated string.)